var/spack/repos/builtin/packages/jsonnet/package.py

```diff
@@ -3,41 +3,79 @@
 # SPDX-License-Identifier: (Apache-2.0 OR MIT)
 
 
-from spack.build_systems.python import PythonPipBuilder
+import spack.build_systems.python
+from spack.build_systems import cmake, makefile
 from spack.package import *
 
 
-class Jsonnet(MakefilePackage):
+class Jsonnet(MakefilePackage, CMakePackage):
     """A data templating language for app and tool developers based on JSON"""
 
     homepage = "https://jsonnet.org/"
     git = "https://github.com/google/jsonnet.git"
     url = "https://github.com/google/jsonnet/archive/refs/tags/v0.18.0.tar.gz"
 
-    maintainers("jcpunk")
+    maintainers("greenc-FNAL", "gartung", "jcpunk", "marcmengel", "marcpaterno")
 
     license("Apache-2.0")
 
     version("master", branch="master")
+    version(
+        "0.21.0-rc1", sha256="0420cc234137ce0cd9314589e22c6e039799ba37b870377b7e3d9b452db8f36e"
```

Comment:
I'm wondering if it makes sense to wait for 0.21.0 rather than add the `rc` release?  Thoughts?

Reply:
`0.20` offers a CMake build, but the 0.21.0-rc1` is the first one that actually works. If you agree, we could mark it `preferred=False`, and that way it would only be utilized if `build_system=cmake` were specified explicitly.

Reply:
I think that probably works for me.

Any guesses when `0.21.0` might get published?  If it is on the order of "weeks" it might be better to wait so we don't have the `rc` version on the list of known versions "forever".

Reply:
`rc1` was tagged 2025-02-22. The only commits to `master` since then have been docs and CI, and the last of those was 9 days ago. I generally remove the RC when adding the main release to the recipe, but it anyway feels to me like the full release will take place in a week or less.

Reply:
Lets give them until Monday.  If by then the real release isn't out we can just merge this as is.

Reply:
Version now updated to 0.21.0